src/App.tsx

```diff
@@ -1,27 +1,86 @@
+import { useState } from 'react';
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import TodoInfo from './components/TodoInfo/TodoInfo';
```

Comment:
Ensure that `TodoInfo` is imported as a default import. Verify that `TodoInfo` is exported as a default export in its module.